### PR TITLE
Updating Sphinx Documentation

### PR DESCRIPTION
**Pull request by reddraconi**
@niftools/blender-niftools-addon-reviewer 

# Overview

1. Lots of minor formatting changes to existing Sphinx documentation. This should clean up most of it.
    (I still need to do a deep-dive to find any broken links or the like.)
2. Added the early version of the "how to install the NifTools Blender Plugin" gif into ``_static/images`` that
    was posted to Discord.gg/NifMania/#blender-plugin earlier today.
3. Updated ``conf.py`` to use ``sphinx.ext.imgmath`` rather than ``sphinx.ext.pngmath`` as the latter has
     been deprecated.
4. Updated ``conf.py`` file to use the default ``bizstyle`` theme, as the ``niftools_sphinx_plugin`` was
    missing from the release v0.0.1 repo.

##  Detailed Description
Documentation update only.

## Fixes Known Issues
No known issues at this time.

## Documentation
Reviewed all Sphinx documentation (*.rst files) to ensure they matched the posted guidelines and
fixed minor grammar issues. Also reformatted long lists (like those seen in the shader pages) to
more concise tables.

## Testing
Sphinx documentation was built using sphinx on Windows 10 and verified via the 
reStructuredText VS Code plugins' built-in linter and the python sphinx module.

### Manual
Review raw files or build with ``make htmldoc``

### Automated
No tests are currently configured for documentation, as far as I can tell, outside of what's
performed by ``make``.

## Additional Information
I don't think I have privileges to branch off of ``/develop``, so here's a PR instead.

### CODE REVIEW

**Comment by reddraconi** [comment]
After chatting with @neomonkeus, I'm going to change the line width on these docs up to 120.
I'll re-comment after those changes are complete.

**Comment by reddraconi** [comment]
Changes complete. Ready for review.

**Comment by reddraconi** [comment]
Reviewed `.rst` files from TagNumElite's repo and made a few minor updates, based on his documentation.
A question came up and was posted in Discord:

1. In docs\user\features\geometry\geometroy.rst (tagnum's update-documentation branch), is the **Vertex Alpha**
section still necessary? This appears to have been combined into the **Vertex Color & Alpha** section of the current documentation.

**Comment by neomonkeus** [comment]
Left some minor comments, its a good first pass as content wise it is probably really out of date in places.
Probably need to be more methodical with PRs and do feature reviews.

File: .gitignore [hunk 1]
```diff
@@ -185,3 +186,5 @@ fabric.properties
 
 # Android studio 3.1+ serialized cache file
 .idea/caches/build_file_checksums.ser
+
+**/blender_niftools_addon-*.zip
```
[the remark this answers is not in the document]
Comment (reddraconi) [reply .gitignore:190]: Fixed, per your input.

File: docs/development/design/index.rst [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 ==============================
 
 .. _development-design:
 
-This section provides a high-level overview of the design of the Blender Niftools Addon.
-It is geared towards anyone interested in delving into developing the Blender Nif Plug-in or knowing more about the development process.
-
+This section provides a high-level overview of the design of the Blender
+Niftools Addon. It is geared towards anyone interested in delving into
+developing the Blender Nif Plug-in or knowing more about the development
```
[the remark this answers is not in the document]
Comment (reddraconi) [reply docs/development/design/index.rst]: Fixed.

File: docs/development/testframework/design/index.rst [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
 TestFramework Design Documentation
 ==================================
 
 .. _developer-testframework-design:
 
 This section provides a high-level overview of the design of the Blender Niftools Addon.
-It is geared towards anyone interested in delving into developing the Blender Nif Plug-in or knowing more about the development process.
-
+It is geared towards anyone interested in delving into developing the Blender
+Nif Plug-in or knowing more about the development process.
```
[the remark this answers is not in the document]
Comment (reddraconi) [reply docs/development/testframework/design/index.rst]: Fixed.

File: CHANGELOG.rst [hunk 4]
```diff
@@ -421,22 +513,22 @@ Version 2.5.0 (22 Nov 2009)
...
-  reported by apwsoft; fix discovered by PacificMorrowind)
+* added preset for Oblivion OL_ANIM_STATIC objects (see issue #2118370 eported
+  by apwsoft; fix discovered by PacificMorrowind)
 
 * export XYZ rotations for object animations instead of converting to
   quaternions (reported by Artorp)
 
-* set the bhkCollosionObject flag to 41 instead of the default 1 for
-  animated (OL_ANIM_STATIC) objects (reported by Artorp)
+* set the bhkCollosionObject flag to 41 instead of the default 1 for aimated
```
[the remark this answers is not in the document]
Comment (reddraconi) [reply CHANGELOG.rst:530]: Fixed.

File: CHANGELOG.rst [hunk 5]
```diff
@@ -765,14 +837,15 @@ Version 2.3.4 (24 Jul 2008)
...
+* removed mopp option from export config dialogue (they are now always
+  generated)
 
 * preserve the "skin", "dynalpha", ... material names
 
 * fixed material merge bug
 
-* fix for nif imports with more than 16 materials per mesh (the materials
-  will not be merged in that case)
+* fix for nif imports with more than 16 materials per mesh (the materialswill
```
[the remark this answers is not in the document]
Comment (reddraconi) [reply CHANGELOG.rst:847]: Fixed.

File: CHANGELOG.rst [hunk 6]
```diff
@@ -1501,13 +1582,16 @@ Version 1.1 (Oct 31, 2005)
 
 * Import: Small fix in the import of vertex colours.
 
-* Import: Autodetect Morrowind style texture path; if you load a NIF from ...\meshes\... then the importer will look in ...\textures\* for the NIF textures.
+* Import: Autodetect Morrowind style texture path; if you load a NIF from
+  ...\meshes\... then the importer will look in ...\textures\* for the NIF
```
[the remark this answers is not in the document]
Comment (reddraconi) [reply CHANGELOG.rst:1586]: Fixed and wrapped with "``"

File: CHANGELOG.rst [hunk 7]
```diff
@@ -4,129 +4,235 @@ Version v0.0.1
...
++========+====================================================================+
+| 361    | Feature: Import Pose                                               |
++--------+--------------------------------------------------------------------+
+| 353    | Automatically select suitable axis orientation                     |
++--------+--------------------------------------------------------------------+
+| 11     | Billboard support                                                  |
++--------+--------------------------------------------------------------------+
+| 15     | Vertex Alpha prop support                                          |
++--------+--------------------------------------------------------------------+
+| 25     | `` NiAlphaProperty`` detection for textures                        |
```
[the remark this answers is not in the document]
Comment (reddraconi) [reply CHANGELOG.rst:27]: I'm not sure what you meant, so I reformatted to 120 cols.
I can revert to the old bulleted format, if you like as well.

File: docs/development/setup/ide.rst [hunk 8]
```diff
@@ -1,58 +1,83 @@
...
+the port as per the util_debug.py settings. This can then be launched and
+should wait for the debug thread to call back into the server.
+
+.. code-block:: shell
 
-In the IDE, create a new configuration of type `Python Debug Server`, update the port as per the util_debug.py settings.
-This can then be launched and should wait for the debug thread to call back into the server.::
+  Starting debug server at port 1234
+  Use the following code to connect to the debugger:
+  import pydevd_pycharm
```
[the remark this answers is not in the document]
Comment (reddraconi) [reply docs/development/setup/ide.rst:59]: Changed from ``shell`` to ``python``.

File: docs/development/design/overview.rst [hunk 9]
```diff
@@ -1,122 +1,136 @@
...
 
+-------------------------
 Development Methodology
------------------------
-
+-------------------------
 A development methodology describes the principles we want as part of the development process.
 
-For the 2.6.x series of development, we decided to develop a Feature-Oriented, Test Driven Development (TDD) methodology to suit both current and future needs.
+For the 2.6.x series of development, we decided to develop a Feature-Oriented, Test Driven Development (TDD)
```
[the remark this answers is not in the document]
Comment (reddraconi) [reply docs/development/design/overview.rst:14]: Updated.

File: docs/development/design/workflow.rst [hunk 10]
```diff
@@ -19,23 +21,21 @@ We are following the following strategy for porting the addon:
 
 The 0.x series is designated as purely experimental.
```
[the remark this answers is not in the document]
Comment (reddraconi) [reply docs/development/design/workflow.rst:22]: I added some TODO comments to address that concern.

**Comment by reddraconi** [comment]
I think I've addressed all of your comments. Please let me know if there's anything else you'd like to be updated, changed, or burned to the ground.

File: CHANGELOG.rst (hunk 7 above)
[the remark this answers is not in the document]
Comment (neomonkeus) [reply CHANGELOG.rst]: Nah was just having some fun, there was a space character between `` Ni

File: docs/development/design/workflow.rst (hunk 10 above)
[the remark this answers is not in the document]
Comment (neomonkeus) [reply docs/development/design/workflow.rst:25]: Yeah definitely out of scope for this PR.

**Review by neomonkeus — APPROVED** [review]

**Pull request merged**